pgcli/main.py

```diff
@@ -181,6 +181,7 @@ def __init__(
         prompt_dsn=None,
         auto_vertical_output=False,
         warn=None,
+        env_name=None,
```

Comment:
It seems excessive to add a new config option just to warn that "You're about to run a destructive command in PROD". You could serve the same purpose by simply printing out the connection string. What do you think?

Reply:
We use ssh-tunnels to connect to all of our environments which makes the connection string usually pointing to localhost at some port. Because of that it's hard to tell which environment we are in and creates a risk of applying a destructive command to an environment we didn't intent to. I know every organization does things differently, but I've seen this problem in different organizations and it would be pretty simple solution to prevent it. Would instead of `env_name` calling it a `alias` or `connection_alias` be better?

Reply:
Are you using dsn aliases?

https://github.com/dbcli/pgcli/blob/366c01bbb97fee96a8169629854af7c83d626c99/pgcli/pgclirc#L200

That might be what you're looking for. You can even display them in prompt:

https://github.com/dbcli/pgcli/blob/366c01bbb97fee96a8169629854af7c83d626c99/pgcli/pgclirc#L141

That way, there's always a reminder of what you're connected to.

Reply:
Omg! That was exactly what I was looking for :) Thank you! For the record, I had no idea that was an option. Why is it called DSN? I stayed away from it because I didn't know. I would suggest naming something easier for people to find and understand. 
I would still think it would be useful to have the destructive warning say which "alias" it is, do you think that's something I could include as part of this PR? if the alias `-D` option is passed?

Reply:
Updated the PR let me know what you think. Thank you!

Reply:
> Omg! That was exactly what I was looking for :) Thank you! For the record, I had no idea that was an option. Why is it called DSN? I stayed away from it because I didn't know. I would suggest naming something easier for people to find and understand.

Oh man, where to start. DSN is an old and pretty widely used term in the database world, it's been around probably since ODBC and JDBC drivers. Could call it "connection string". Maybe the original PR author thought that `connection_string_alias` is too long for a name. The `alias_dsn` section in config could use better comments. Feel free to add some.

> I would still think it would be useful to have the destructive warning say which "alias" it is, do you think that's something I could include as part of this PR? if the alias -D option is passed?

Yes, I believe that would be useful.

Reply:
Ah, that makes sense. Thank you for explaining. I'll add some comments.